autorest/codegen/templates/templates_package_dataplane/dev_requirements.txt.jinja2

```diff
@@ -0,0 +1,7 @@
+-e ../../../tools/azure-devtools
+-e ../../../tools/azure-sdk-tools
+../../core/azure-core
+../../identity/azure-identity
```

Comment:
we don't know if we need identity. Also we don't need to do 3.x checks anymore, I think by the time we merge this pr, we will have dropped 2.x support in autorest